### ISSUE TITLE
Deprecate Pure_parallels caveat example

### ISSUE THREAD

**Issue by drlaw1558** [spacetelescope/jwst-caveat-examples#18]
The issue addressed by the pure_parallels caveat notebook has been fixed in Build 10.2, and the notebook can thus be deprecated.

**Comment by drlaw1558** [comment]
Tagging @sosey @goudfroo

**Comment by goudfroo** [comment]
Oh wow, that's quicker than I expected, thanks for letting us know. What is the relevant ticket?

**Comment by drlaw1558** [comment]
Hi Paul- https://jira.stsci.edu/browse/JSDP-2816 should be the final ticket that closed the issue, with links to various other tickets involved in the problem (including your original JWSTSOCINTDR ticket).

**Comment by goudfroo** [comment]
Excellent, thank you. I propose to keep the script available, namely for folks who already downloaded pure parallel data prior to Build 10.2. I will add relevant info in the README.

**Comment by sosey** [comment]
the deprecation process is on the front page of the repo, let's start there....does the 3 month timeframe work for you @goudfroo ? If the script should be around long term, we should consider moving the information to a notebook in JDAT.

**Comment by goudfroo** [comment]
Hi @sosey - thanks for that info. That timeframe should work in principle, I'll have to see how to transform it into a notebook. First we'll have to wait to find out when the relevant data in MAST have been reprocessed with Build 10.2.

**Comment by sosey** [comment]
Yup, reprocessing timeframe is part of the deprecation process. If we need to update the process on this repo to accommodate scripts we can... I would say comment heavily in the script itself as well as the README file in that directory to inform users and make sure the rest of the items in the list are taken care of as well.

**Comment by sosey** [comment]
where are we at with this @goudfroo ?

**Comment by sosey** [comment]
@goudfroo reping :)

**Comment by goudfroo** [comment]
I put in a pull request to delete the Pure_Parallel directory 2 months ago. Looks like I asked you and Bryan Hilbert to review that pull request (since Bryan is listed as the official owner).

**Comment by sosey** [comment]
merged
